apps/docs/src/app/core/component-docs/feed-list-item/examples/fli-action/fli-action-example.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="fd-feed-list" aria-label="Feed List Item with user link example">
+    <fd-feed-list-item
+        authorTitle="John Doe"
+        authorLink="http://example.com/"
+        text="<h4>Long text, you can toggle</h4> view with more/less button. Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+        maxRows="2"
+        ariaLabel="Feed list item example">
+        <ng-template #avatar>
+            <div class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link">
+                <img
+                    aria-label="John Doe"
+                    title="John Doe"
+                    src="https://avatars.dicebear.com/api/human/fe.svg?m=6&b=%23f5f0f0&mood[]=surprised">
+            </div>
+        </ng-template>
+        <ng-template #action>
+            <button aria-label="action" fd-button fdType="transparent" [compact]="true" glyph="overflow"></button>
+        </ng-template>
+        <ng-template #footer>
```

Comment:
I think for consistency with other components, rather than using ng-template in the docs here, instead the feed list footer/avatar/action should get their own components.  And fd-feed-list will be a component. So it'd look something like this
```
<fd-feed-list>
  <fd-feed-list-item>
    <fd-feed-list-avatar>...</fd-feed-list-avatar>
    <fd-feed-list-action>...</fd-feed-list-action>
    <fd-feed-list-footer>...</fd-feed-list-footer>
  </fd-feed-list-item>
<fd-feed-list>
```